frontend_tests/node_tests/poll_widget.js

```diff
@@ -21,7 +21,7 @@ run_test("PollData my question", () => {
     const sender_id = 99;
     people.my_current_user_id = () => sender_id;
 
-    const data_holder = new poll_widget.PollData(is_my_poll, question, []);
+    const data_holder = new poll_widget.PollData(is_my_poll, question, [], blueslip.warn);
```

Comment:
We could also think about having the mobile app provide a `blueslip` module that just converts the interface; this is kinda the sort of thing that one wants to be able to add in random low-level code without thinking about plumbing something down to clients.

Reply:
As a data point, there's a ton of these and they're conceptually the kind of thing that's going to be scattered all over otherwise shareable modules.
```
$ git grep blueslip[.] | wc
    455    2992   42735
```